packages/components/utils/src/useDensity/useDensity.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: 'useDensity'
+type: 'hook'
+slug: /hooks/use-density/
+section: 'hooks'
+typescript: ./useDensity.ts
+---
+
+Allows to retrieve the `DensityContext` value (`low` or `high`).  
```

Comment:
Should we point to the `DensityContainer` documentation??

Reply:
It is mentioned in the examples below 🙂